pandas/core/common.py

```diff
@@ -1734,7 +1735,7 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
                                     bounds_error=bounds_error)
```

Comment:
Should pass ``**kwargs``, or  only support ``UnivariateSpline``? (Also #1749line)